corehq/apps/app_manager/templates/app_manager/partials/modules/module_view_settings.html

```diff
@@ -267,6 +267,22 @@ <h4 class="panel-title panel-title-nolink">{% trans "Advanced" %}</h4>
               </div>
             {% endif %}
 
+            {% if request|toggle_enabled:'CASE_LIST_OPTIMIZATIONS' and not module.is_surveys %}
+              <div class="form-group">
+                <label class="control-label {% css_label_class %}">
+                  {% trans "Show Case List Optimizations" %}
+                  <span class="hq-help-template"
+                        data-title="{% trans "Case List Optimizations" %}"
```

Comment:
Are there docs on this feature to link to? This seems like the kind of feature where users are going to want a reference to exactly what the options mean.

Reply:
Documents are yet to be created.

This is pretty early work to get things on HQ proactively. The mobile changes are yet to be done after which the documentation will be added and only once the new CC app is released, this would actually be a usable feature. So, documentation is on the way!